envoy/network/drain_decision.h

```diff
@@ -1,19 +1,37 @@
 #pragma once
 
+#include <chrono>
+#include <functional>
+
+#include "envoy/common/callback.h"
 #include "envoy/common/pure.h"
 
+#include "absl/base/attributes.h"
+
 namespace Envoy {
 namespace Network {
 
 class DrainDecision {
 public:
+  using DrainCloseCb = std::function<void(std::chrono::milliseconds)>;
```

Comment:
Can you add comments here on what the milliseconds param means? It's not immediately clear to the reader.

Reply:
Added in e568d7b907712df3e034cc65887bbc185ea51563 on #16201